jans-cedarling/bindings/cedarling-java/README.md

```
@@ -72,25 +69,96 @@ To use Cedarling Java bindings in Java Maven Project add following `repository`
-| ACTION_FILE_PATH     | ./custom/static/action.txt      |
-| RESOURCE_FILE_PATH   | ./custom/static/resource.json   |
-| CONTEXT_FILE_PATH    | ./custom/static/context.json    |
-| PRINCIPALS_FILE_PATH | ./custom/static/principals.json |
+    | Key                  | Values                          |
+    | -------------------- | ------------------------------- |
+    | BOOTSTRAP_JSON_PATH  | ./custom/static/bootstrap.json  |
 
 4. Map the script with client used to perform authentication.
 
-![](./docs/mapping_post_authn_script_with_client.png)
+![](./docs/mapping_update_token_script_with_client.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)